learn/guides/asyncapi-tool.md

```diff
@@ -16,15 +16,101 @@ The Ballerina AsyncAPI tool will make it easy for you to start the development o
 
 > **Prerequisite:** Install the latest [Ballerina Swan Lake distribution](https://ballerina.io/downloads/).
```

Comment:
We better follow a similar approach to other guides when giving prerequisites. See https://ballerina.io/learn/get-started-with-ballerina/

And since this was introduced later in SL , mention the exact version such as `[Ballerina 2202.1.0 (Swan Lake)](https://ballerina.io/learn/installing-ballerina/setting-up-ballerina/) or greater`